static-site/content/blog/2024-10-22-oropouche-analysis-and-resources.md

```diff
@@ -0,0 +1,66 @@
+- [https://nextstrain.org/oropouche/L](https://nextstrain.org/oropouche/L)
+- [https://nextstrain.org/oropouche/M](https://nextstrain.org/oropouche/M)
+- [https://nextstrain.org/oropouche/S](https://nextstrain.org/oropouche/S)
+
+The segmented nature of Oropouche virus allows for genetic reassortment events. Similarly to [Gutierrez et al.](https://pmc.ncbi.nlm.nih.gov/articles/PMC7022353/), all three phylogenies display topographical differences, suggesting the presence of reassortment events on the genome. For example, while the phylogenies of the L and M segments show more robust clustering of the sequences from Brazil throughout time, the S segment phylogeny shows the Brazil sequences scattered throughout the tree  (**Fig. 4**). If we focus on the most recent sequences from 2022-2024, however, we do find more robust clustering of the sequences from Brazil, similar to the results presented by [Naveca et al](https://www.nature.com/articles/s41591-024-03300-3).
+
+[![fig4](img/oropouche_s_and_m_segment.png)](https://nextstrain.org/oropouche/M)
+**Figure 4. Comparison of the M (top) and S (bottom) segment trees shows distinct topographical differences, suggesting the presence of reassortment events.** Available as [nextstrain.org/oropouche/M](https://nextstrain.org/oropouche/M) and [nextstrain.org/oropouche/S](https://nextstrain.org/oropouche/S) respectively.
+
+We can track the co-evolution of the Oropouche segments via tanglegrams. For example, if we focus on the L and M segments (**Fig. 5**), we can see evidence of at least two recombination events.
+
+[![fig5](img/oropouche_tanglegram.png)](https://nextstrain.org/oropouche/L:oropouche/M)
+**Figure 5. Tanglegram of L and M segments.** Available as [nextstrain.org/oropouche/L:oropouche/M](https://nextstrain.org/oropouche/L:oropouche/M).
+
+# Nextstrain resources
+
+## Curated sequences and metadata
+
+We curate sequence data and metadata from NCBI as the starting point for our analyses. Curated sequences and metadata are available as flat files at:
+
+L segment
+
+- [data.nextstrain.org/files/workflows/oropouche/L/sequences.fasta.zst](data.nextstrain.org/files/workflows/oropouche/L/sequences.fasta.zst)
+- [data.nextstrain.org/files/workflows/oropouche/L/metadata.tsv.zst](data.nextstrain.org/files/workflows/oropouche/L/metadata.tsv.zst)
+
+M segment
+
+- [data.nextstrain.org/files/workflows/oropouche/M/sequences.fasta.zst](data.nextstrain.org/files/workflows/oropouche/M/sequences.fasta.zst)
+- [data.nextstrain.org/files/workflows/oropouche/M/metadata.tsv.zst](data.nextstrain.org/files/workflows/oropouche/M/metadata.tsv.zst)
+
+S segment
+
+- [data.nextstrain.org/files/workflows/oropouche/S/sequences.fasta.zst](data.nextstrain.org/files/workflows/oropouche/S/sequences.fasta.zst)
+- [data.nextstrain.org/files/workflows/oropouche/S/metadata.tsv.zst](data.nextstrain.org/files/workflows/oropouche/S/metadata.tsv.zst
+)
```

Comment:
_Totally missed this PR_ 🙈 

The structure of the underlying data changed to 1 metadata TSV + 3 FASTAs in https://github.com/nextstrain/oropouche/pull/18. 

See the latest description at https://github.com/nextstrain/oropouche/blob/main/phylogenetic/defaults/description.md#underlying-data

Reply:
(I was wondering why the [live build](https://next.nextstrain.org/oropouche/L) footer hasn't been updated and it's due to the [cache not updating](https://github.com/nextstrain/oropouche/issues/20))

Reply:
Links were updated in https://github.com/nextstrain/nextstrain.org/pull/1046